specification/configuration.md

```diff
@@ -227,12 +228,23 @@ are required.
     **NOTE: `jaeger-thrift-splunk` is deprecated.**
     If the user selects `jaeger-thrift-splunk`, distributions MUST log a deprecation warning and suggest an alternate method. For example:
 
-    ```
+    ```log
     jaeger-thrift-splunk trace exporter is deprecated and may be removed in a future major release. Use the default 
     OTLP exporter instead, or set the SPLUNK_REALM and SPLUNK_ACCESS_TOKEN environment variables to send 
     telemetry directly to Splunk Observability Cloud.
     ```
 
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+The following environment variables are NOT required
+(the instrumentation libraries MAY support them):
```

Comment:
```suggestion
(instrumentation libraries MAY support them):
```

Reply:
@breedx-splk are these "workaround" settings or something that eventually should reach the GDI spec?

Reply:
I'm having second thoughts if we want to document such config options here. Unless this is something that we think that eventually, all instrumentation should have, it will be easier to just remove them.

Reply:
Per @mateuszrzeszutek comment: we do not want to have these settings. I reverted the PR to the original state.